### PR TITLE
Dev client

### PR DESCRIPTION
**Pull request by OlegGre4ka**

### CODE REVIEW

File: web-client/README.md [hunk 1]
```diff
@@ -1,46 +0,0 @@
-# Getting Started with Create React App
```
Comment (mrPronin) [inline web-client/README.md:1]: It would be better to have a README file for the project, but update it.

File: web-client/public/manifest.json [hunk 2]
```diff
@@ -1,25 +0,0 @@
-{
```
Comment (mrPronin) [inline web-client/public/manifest.json:1]: I also see that the files for PWA support have been removed. Was this done on purpose?

File: web-client/src/App.test.tsx [hunk 3]
```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```
Comment (mrPronin) [inline web-client/src/App.test.tsx:1]: It is recommended to add tests for the existing code.

**Review by mrPronin — CHANGES_REQUESTED** [review]

File: web-client/README.md (hunk 1 above)
Comment (OlegGre4ka) [reply web-client/README.md:1]: Done. With the comands for run and build the app - npm run dev, npm run build

File: web-client/public/manifest.json (hunk 2 above)
Comment (OlegGre4ka) [reply web-client/public/manifest.json:1]: Done, Installed vite-plugin-pwa, and added PWA manifest in the vite.config.ts

File: web-client/src/App.test.tsx (hunk 3 above)
Comment (OlegGre4ka) [reply web-client/src/App.test.tsx:1]: I am going to add tests later, after learning with this topic

**Review by mrPronin — APPROVED** [review]

**Pull request merged**